stablehlo/reference/Element.h

```diff
@@ -42,6 +42,7 @@ class Element {
   Element(Type type, std::complex<APFloat> value);
   Element(Type type, std::complex<double> value);
   Element(const Element &other) = default;
+  Element() = default;
```

Comment:
If we chose to go the route suggested in the comment above, we will no longer need to add this.